scripts/bump-version/index.js

```diff
@@ -99,7 +99,7 @@ const updatePeerDeps = (pkgJson, newMonorepoVersion) => {
     Object.keys(peerDependencies).forEach((dep) => {
         if (monorepoPackageNames.includes(dep)) {
             console.log(`Found lerna local package ${dep} as a peer dependency of ${pkgJson.name}.`)
-            peerDependencies[dep] = `^${newMonorepoVersion}`
+            peerDependencies[dep] = newMonorepoVersion
```

Comment:
Oh one thing worth mentioning: `syncpack` expected the peer dependencies in this case to be _exact_ version. So I had to update our script as a result.

Reply:
@vmarta just FYI, If you ever need to change it you can configure syncpack to use a https://jamiemason.github.io/syncpack/config/semver-groups so that peerDependencies always have a `^`